sdk/storage/examples/blob_04.rs

```diff
@@ -30,12 +31,12 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     let data = b"1337 azure blob test";
     let mut block_ids = Vec::new();
     for (i, block) in data.chunks(64 * 1024 * 1024 /* 64 MiB */).enumerate() {
-        block_ids.push(i.to_be_bytes());
+        block_ids.push(Bytes::from(format!("{}", i)));
```

Comment:
If these are bytes, may be use a BytesMut. https://docs.rs/bytes. If I'm reading this wrong, please ignore.

Reply:
Bytes also supports `chunks`. https://docs.rs/bytes/1.0.1/bytes/struct.Bytes.html#method.chunks if you make `data` a Bytes.

Reply:
Yes you are right, the example should be more meaningful (sending a block list with just one block is silly 🤦‍♂️ ). I've updated the example to show `BytesMut`, `freeze` and proper chunking.